swap/src/protocol/bob/swap_request.rs

```diff
@@ -35,14 +35,14 @@ pub struct OutEvent {
 #[behaviour(out_event = "OutEvent", poll_method = "poll")]
```

Comment:
Why is this not using `event_process = false`?

Reply:
Because we are not all as verse as you with libp2p :)

Reply:
Time to change that! :D